app/models/user.rb

```diff
@@ -1,8 +1,8 @@
 class User < ApplicationRecord
   acts_as_token_authenticatable
   # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :trackable, :validatable
+  # :confirmable, :lockable, :timeoutable and :omniauthable, :registerable
```

Comment:
Can't find info on the interwebs what "registerable" means.

Reply:
registerable -> https://github.com/plataformatec/devise/blob/master/lib/devise/models/registerable.rb

Reply:
i disable that so people don't just go to the sign up page and signup for an account.

Reply:
How do you create user accounts?

In particular, if I check out this branch and pull the dev db, will I be locked out?

Reply:
I am guessing the way this will work is that we manually insert user entries in the SQL DB and then the user follows the "i forgot my password" link to reset it?

Reply:
we will manually enter one through "rails c" and i will enable inviteable so you can invite people if necessary.